waka-worker/stats/index.js

```diff
@@ -0,0 +1,16 @@
+class Stats {
```

Comment:
I'm fairly sure that this information is already exposed, cause this works: https://assets-us-west-2.waka.app/status/index.html